CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Added
+- Support for `images` on `TimelineMarkers`
+
+### Changed
+- The way `TimelineMarkers` are rendered.
```

Comment:
Is this really relevant information as long as the functionality stays the same?

Reply:
Since it is open source, I wanted to add this to the changelog. So if anyone did some custom implementation around this, can catch it in time.

Reply:
Good point, I didn't consider which custom implementations might rely on this :+1:

Reply:
@TeslaAdis I guess you missed that comment.

Reply:
Indeed I did

Reply:
Addressed in https://github.com/bitmovin/bitmovin-player-ui/pull/318/commits/a112ec2396b28ee9cb6b7c9f7049498f07924378